tcms/bugs/tests/test_permissions.py

```diff
@@ -58,6 +59,32 @@ def verify_post_with_permission(self):
         self.assertContains(response, 'BUG-%d' % last_bug.pk)
 
 
+class TestEdit(tests.PermissionsTestCase):
+    permission_label = 'bugs.change_bug'
+    http_method_names = ['get', 'post']
+
+    @classmethod
+    def setUpTestData(cls):
+        bug = BugFactory()
+
+        cls.url = reverse('bugs-edit', args=(bug.pk,))
+        cls.post_data = {
+            'summary': 'An edited summary'
+        }
+
+        super().setUpTestData()
+
+    def verify_get_with_permission(self):
+        response = self.client.get(self.url)
+
+        self.assertContains(response, _('Edit bug'))
+        self.assertContains(response, 'bugs/js/mutable.js')
+
+    def verify_post_with_permission(self):
+        response = self.client.post(self.url, self.post_data, follow=True)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Status 200 doesn't tell me on which page I am. Upon successful change the test case verify that the new summary is shown on the page, also can check the bug number - see how the page looks in real life.